### ISSUE TITLE
Header keyword coloring bug?

### ISSUE THREAD

**Issue by fonnesbeck** [fletcher/markdown.tmbundle#6]
The MMD bundle does not seem to catch all of the header keywords from the sample document:

http://cl.ly/2D382L0H2K351o2z2M42

**Comment by fletcher** [comment]
So as not to make false promises - I probably won't get around to fixing this.  If someone else comes up with a fix, I'm happy to incorporate it. But it's probably number "last" on my list of things to do....

But I'll certainly leave the issue open until it's resolved.

**Comment by fletcher** [comment]
Is this not the same issue you opened up previously?

https://github.com/fletcher/markdown.tmbundle/issues/2

I thought when I looked into it then, it was a problem with your theme.

**Comment by fletcher** [comment]
But in any event, I'm not particularly interested in tweaking the language parser for the textmate bundle, as I don't find it particularly useful.

Perhaps someone who does make heavy use of the various scopes would be interested in improving the parser?

**Comment by fonnesbeck** [comment]
Yeah, that was me, sorry. I had forgotten that I had posted it elsewhere, and did not see it among the open issues.

Its not a bug specific to a particular theme, no (at least, it occurs with all the themes that I have installed). If I come up with a fix for the bundle, I will issue a pull request.

**Comment by fletcher** [comment]
It might be that there were two separate issues???

In any case, I look forward to what you come up with.
